test/db/cmd/feat_grep

```diff
@@ -165,10 +165,10 @@ i~:..3
 i~:1..2
 i~:-2
 i~:-1
-?e 0\n1\n2\n3\n4~:-2
-?e 0\n1\n2\n3\n4~:-2..
-?e 0\n1\n2~:0..
-?e 0\n1\n2~:..
+echo "0\n1\n2\n3\n4"~:-2
+echo "0\n1\n2\n3\n4"~:-2..
+echo "0\n1\n2"~:0..
+echo "0\n1\n2"~:..
```

Comment:
TODO: check if `echo` unescape characters like `?e` did.

Reply:
No that doesn't work, because I replaced the old echo code with code in `?e` command in order to be able to replace flags in string like the `?e` command did. Only then I could have removed `?e` command group and kept from failing lots of regression tests.

Reply:
Sorry, what do you mean? which places are you going to replace?

Reply:
I mean I'll remove `"` where it's present with the `echo` command and not needed (as your TODO states)